osa/scripts/gain_selection.py

```diff
@@ -123,27 +125,47 @@ def apply_gain_selection(date: str, output_basedir: Path = None):
         
         for file in r0_files:
             output_file = output_dir / file.name
-            file.link_to(output_file)
+            sp.run(["cp", file, output_dir])
+
+def run_sacct_j(job) -> StringIO:
```

Comment:
There should be a similar function in `osa.job`. Also, you'll probably have to take into account the `starttime` option in `sacct` since by default it only retrieves the information from the last 24 hours or even from the current day.

Reply:
The idea is to call first `run_sacct` and use its output in `get_sacct_output`. Perhaps this could be better within a unique class. But that is a different matter.

Reply:
I also use the function get_sacct_output below, and it is not necessary to give the `starttime` option because I set the job_id.

Reply:
Got it